tap_github/comments.json

```
@@ -0,0 +1,97 @@
+{
+  "type": ["object"],
```

Sorry, I know that the two of these are semantically equivalent but this should be, conventionally, exactly `"type": "object"`, without the surrounding square brackets.

```suggestion
  "type": "object",
```